test/google_crawler_web/views/api/auth_view_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule GoogleCrawlerWeb.Api.AuthViewTest do
+  use GoogleCrawlerWeb.ViewCase
```

Comment:
Just 1 small comment @hanam1ni I approved the PR but could you double-check all-new test files in this PR, can we add the `, async: true` to it? I commented on a few test files before but you could also check and add the `, async: true` to all test files

Reply:
@andyduong1920 Sure 👍  I will update in this PR